charts/newrelic-prometheus-agent/values.yaml

```diff
@@ -182,8 +182,21 @@ config:
   # -- It allows defining scrape jobs for Kubernetes in a simple way.
   # @default -- See `values.yaml`
   kubernetes:
+
+    # By default, all targets discovered are scraped.
+    # If curated_experience is enabled or any regex is specified,
```

Comment:
is this right? if an entry is added to `additional_regexes` the `curated_experience` is activated?

Reply:
Pehaps something like this :
NR has a list of Dashboards, alerts, ... for a number of Services , the curated_experience allows to filter targets which NR has this experience out of the box. The filter is based on the target kubernetes labels which are pre-defined and can be extended by this configuration...

Reply:
Added that!